### PR TITLE
New package: gnome-authenticator-0.2.4

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: srcpkgs/python-pyotp/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	python -B test.py
+}
+
+post_install() {
+	vlicense LICENSE
+}
+
+python3-pyotp_package() {
+	noarch=yes
+	pycompile_module="pyotp"
```
Comment (maxice8) [inline srcpkgs/python-pyotp/template]: short_desc should be the same as the one with Python replaced with Python3

File: srcpkgs/python-pyzbar/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+build_style=python-module
+pycompile_module="pyzbar"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="libzbar python-enum34"
+short_desc="Read one-dimensional barcodes and QR codes"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+license="MIT"
+homepage="https://github.com/NaturalHistoryMuseum/pyzbar/"
+distfiles="https://github.com/NaturalHistoryMuseum/pyzbar/archive/v${version}.tar.gz"
+checksum=4dbbece533650f2aeb6c8d1f41cf424614d2877d7331c48a9eed35ae9f949626
```
Comment (maxice8) [inline srcpkgs/python-pyzbar/template]: is problably missing alternatives=

Comment (Cogitri) [reply srcpkgs/python-pyzbar/template]: Ah, yeah, thanks

File: srcpkgs/python-pyotp/template [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+do_check() {
+	python -B test.py
+}
+
+post_install() {
+	vlicense LICENSE
+}
+
+python3-pyotp_package() {
+	short_desc="Python3 One-Time Password Library"
```
Comment (maxice8) [inline srcpkgs/python-pyotp/template]: you cna just short_desc="${short_desc/Python2/Python3}"

Comment (Cogitri) [reply srcpkgs/python-pyotp/template]: Hm, I personally prefer keeping it simple by not doing stuff like that, I'll change it, seems like idiomatic Void

File: srcpkgs/python-pyotp/template [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	vlicense LICENSE
+}
+
+python3-pyotp_package() {
+	short_desc="${short_desc/Python2/Python3}"
+	noarch=yes
+	pycompile_module="pyotp"
+	depends="python3"
+	pkg_install() {
+		vmove usr/lib/python3*
```
Comment (asergi) [inline srcpkgs/python-pyotp/template]: `vlicense LICENSE` here too.

File: srcpkgs/python-pyzbar/template [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# Template file for 'python-pyzbar'
+pkgname=python-pyzbar
+version=0.1.7
+revision=2
```
Comment (asergi) [inline srcpkgs/python-pyzbar/template]: `revision=1` for new packages.

File: srcpkgs/python-pyzbar/template [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+noarch=yes
+wrksrc="pyzbar-${version}"
+build_style=python-module
+pycompile_module="pyzbar"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="libzbar python-enum34 python-dateutil"
```
Comment (asergi) [inline srcpkgs/python-pyzbar/template]: - `dateutil` is not required
- `depends="libzbar python-enum34 python-pathlib"`

Note: `libzbargtk` and `libzbarqt` should be splitted out of `libzbar` to minimize dependencies.
Also, the Python2 bindings provided by the `zbar` package should be splitted as well or removed.

File: srcpkgs/python-pyzbar/template [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+short_desc="Read one-dimensional barcodes and QR codes"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+license="MIT"
+homepage="https://github.com/NaturalHistoryMuseum/pyzbar/"
+distfiles="https://github.com/NaturalHistoryMuseum/pyzbar/archive/v${version}.tar.gz"
+checksum=4dbbece533650f2aeb6c8d1f41cf424614d2877d7331c48a9eed35ae9f949626
+alternatives="read_zbar:/usr/bin/read_zbar:/usr/bin/read_zbar.py2"
```
Comment (asergi) [inline srcpkgs/python-pyzbar/template]: Wrong: `alternatives="pyzbar:read_zbar:/usr/bin/read_zbar.py2"`.

File: srcpkgs/python-pyzbar/template [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+	vlicense LICENSE.txt
+	# These only exec read_zbar.py{2,3}
+	rm "$DESTDIR"/usr/bin/read_zbar{2,3}
+}
+
+python3-pyzbar_package() {
+	noarch=yes
+	pycompile_module="pyzbar"
+	depends="libzbar python3-dateutil"
+	alternatives="read_zbar:/usr/bin/read_zbar:/usr/bin/read_zbar.py3"
```
Comment (asergi) [inline srcpkgs/python-pyzbar/template]: - ditto for `dateutil` and `alternatives`
- `depends="libzbar python3"`

File: srcpkgs/python-pyzbar/template [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+	pkg_install() {
+		vmove usr/lib/python3*
+		vmove usr/bin/read_zbar.py3
```
Comment (asergi) [inline srcpkgs/python-pyzbar/template]: `vlicense LICENSE.txt` here too.

File: srcpkgs/gnome-authenticator/template [hunk 10]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'gnome-authenticator'
+pkgname=gnome-authenticator
+version=0.2.4
+revision=1
+_libgd_hash=7ae254bfc5f641c60566614e08245176f7bc5aa8
+wrksrc="Authenticator-${version}"
+build_style=meson
```
Comment (asergi) [inline srcpkgs/gnome-authenticator/template]: `pycompile_moudle="Authenticator"`

File: srcpkgs/python-pyotp/template [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+revision=1
+noarch=yes
+wrksrc="pyotp-${version}"
+build_style=python-module
+pycompile_module="pyotp"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python"
+short_desc="Python2 One-Time Password Library"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+license="0BSD"
```
[the remark this answers is not in the document]
Comment (Cogitri) [reply srcpkgs/python-pyotp/template]: Ah, seems like it switched to it, it's still listed as BSD on https://pypi.org/project/pyotp/

**Pull request merged**